pkg/helpers/newapp/newapptest/newapp_test.go

```diff
@@ -759,7 +759,7 @@ func TestNewAppRunAll(t *testing.T) {
 				Resolvers: cmd.Resolvers{
 					DockerSearcher: app.DockerClientSearcher{
 						Client: &apptest.FakeDockerClient{
-							Images: []docker.APIImages{{RepoTags: []string{"centos/ruby-27-centos7"}}},
+							Images: []docker.APIImages{{RepoTags: []string{"quay.io/centos7/ruby-27-centos7"}}},
```

Comment:
Hmmm... something still looks off, especially that we use that `centos/ruby-27-centos7` image in a previous test case. I'll check it more thoroughly on Monday.

Reply:
Found the change that triggered this https://github.com/openshift/ruby-hello-world/commit/c90a10925b5f532fb9d85afedf98d54547906869 which makes this valid update.